packages/@aws-cdk/aws-cloud9/lib/environment.ts

```diff
@@ -0,0 +1,134 @@
+import * as ec2 from '@aws-cdk/aws-ec2';
+import * as cdk from '@aws-cdk/core';
+import { CfnEnvironmentEC2 } from '../lib/cloud9.generated';
+
+/**
+ * A Cloud9 Environment
+ */
+export interface IEnvironmentEC2 extends cdk.IResource {
```

Comment:
What do you think about a slight name change - `IEc2Environment` instead of `IEnvironmentEc2`? It would also mean changing the name of the properties (like `environmentEc2Name` should be renamed to `ec2EnvironmentName`).

Let me know what you think!

Reply:
Appreciated.

I agree but actually there is a `CfnEnvironmentEC2`, to make it consistent I created `IEnvironmentEc2` for it.

Looks like I still can name it `Ec2Environment` but will encounter the awslint errors:

<img width="777" alt="圖片" src="https://user-images.githubusercontent.com/278432/76583024-06bc2900-6513-11ea-8090-5597491bf0a6.png">

And  looks like I can't get around the last error

```
error: [awslint:props-physical-name:@aws-cdk/aws-cloud9.Ec2EnvironmentProps] Every Resource must have a single physical name construction property, with a name that is an ending substring of <cfnResource>Name 
```

It's kind like I must provide a `environmentEC2Name` property to pass the awslint test.

What can I do for that?

Reply:
You can turn off that linter error by adding an entry in the `package.json` of the module. [Here's an example turning off that error in a different module](https://github.com/aws/aws-cdk/blob/972125cbdc1c3d1cc14310bba1a6abd8a7b8ddb1/packages/%40aws-cdk/aws-codebuild/package.json#L207-L209).